base/cadvisor/README.md

```diff
@@ -0,0 +1,5 @@
+# cAdvisor
+
+[cAdvisor](https://github.com/google/cadvisor) provides container users an understanding of the resource usage and performance characteristics of their running containers. It is a running daemon that collects, aggregates, processes, and exports information about running containers.
+
+cAdvisor is part of the default Sourcegraph cluster installation, and deployed as a [Kubernetes DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/). This setup is based on the [official cAdvisor Kubernetes Daemonset configuration](https://github.com/google/cadvisor/tree/master/deploy/kubernetes).
```

Comment:
We should add a short paragraph below here to the effect of "we use our own cadvisor deployment not the metrics k8s exports itself because it is often outdated" or something like that

Reply:
added a paragraph in 867e1582214a838bfba07f0ae9625fe716cb76da !

Reply:
I think GitHub is breaking? it doesn't show this commit in your PR :/

Reply:
I think it's pushed, should show up eventually 😅